stytch/b2c/session.go

```diff
@@ -20,20 +22,8 @@ type SessionsGetJWKSParams struct {
 }
 
 type SessionsGetJWKSResponse struct {
-	RequestID string `json:"request_id,omitempty"`
-	Keys      []Key  `json:"keys,omitempty"`
-}
-
-type Key struct {
-	Typ     string   `json:"kty"`
-	Use     string   `json:"use"`
-	KeyOps  []string `json:"key_ops"`
-	Alg     string   `json:"alg"`
-	KeyID   string   `json:"kid"`
-	X5C     []string `json:"x5c"`
-	X5TS256 string   `json:"x5tS256"`
-	N       string   `json:"n"`
-	E       string   `json:"e"`
+	RequestID string       `json:"request_id,omitempty"`
+	Keys      []shared.Key `json:"keys,omitempty"`
 }
```

Comment:
Removing this definition is a breaking change. But we can fix that with a re-export:
```go
type Key = shared.Key
```

Up to you if you want to drop a `// Deprecated: Use shared.Key` instead, but I wouldn't want to make someone pull in a new import for just this one type.